api/cmd/ui_handler.go

```diff
@@ -0,0 +1,77 @@
+package main
```

Comment:
Does it make sense to have separate folder for cmd.go, setup.go, and ui_handler.go ? I.e. under cmd/api/

Reply:
yes make sense